### PR TITLE
Update docs on enable_reconfigure now that gcloud bug is fixed

### PR DESCRIPTION
**Pull request by nick-stroud**
Tested:
- Add partition
- Resize partition
- Add network storage
- Delete partition
- gcloud project config mismatch

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md [hunk 1]
```diff
@@ -78,27 +90,10 @@ This option has some additional requirements:
   development environment deploying the cluster. One can use following commands:
 
   ```bash
-  wget https://raw.githubusercontent.com/SchedMD/slurm-gcp/5.7.2/scripts/requirements.txt
-  pip3 install -r requirements.txt --user
+  pip3 install -r https://raw.githubusercontent.com/SchedMD/slurm-gcp/5.7.2/scripts/requirements.txt --user
```
Comment (mr0re1) [inline community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: `--user` will throw if is run from virtualenv. Do we want to discourage usage of virtualenv?

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md [hunk 2]
```diff
@@ -67,6 +67,18 @@ activated through the `enable_reconfigure` setting:
...
+1. Edit the blueprint with the desired configuration changes
+1. Call `ghpc create <blueprint> -w` to overwrite the deployment directory
+1. Follow instructions in terminal to deploy
+
+The following are examples of updates that can be made to a running cluster:
+
+* Add or remove a partition to the cluster
+* Resize an existing partition
+* Attach new network storage to an existing partition
+
```
Comment (cboneti) [inline community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:81]: Optional: maybe add that  changing the VM type of a partition will likely not work. (I tried with a gpu vm today and it did not work -- as it fails to update the instance template). It seems like a common case.

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md (hunk 1 above)
Comment (cboneti) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: good point, also, it doesn't seem necessary if ran by non-root. It default to --user, right?

**Review by cboneti — APPROVED** [review]
I left two comments, consider it approved after you address them.

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md (hunk 1 above)
Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: I am reading that "pip defaults to installing Python packages to a system directory". I went through this process when testing and indeed the `--user` argument is denied when installing into virtualenv, but the error message is very clear. I would opt to leave it as we do not mention anything about a virtual environment. I also don't feel strongly so if you still disagree I can update.

Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: In fact I just ended up removing `--user` as it does not make much difference to me and I want to move forward with this PR without waiting for re-review.

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md (hunk 2 above)
Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:81]: Done.

**Pull request merged**